<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormCierreCaja</class>
 <widget class="QWidget" name="FormCierreCaja">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>547</width>
    <height>338</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="LCaja">
       <property name="text">
        <string>Caja:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="CBCaja">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="LComputado">
       <property name="text">
        <string>Saldo Computado:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="EDSBPrecio" name="dSBComputado">
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="prefix">
        <string>$  </string>
       </property>
       <property name="maximum">
        <double>99999999.989999994635582</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="LSumado">
       <property name="text">
        <string>Saldo Sumado:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="EDSBPrecio" name="dSBSumado">
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="prefix">
        <string>$  </string>
       </property>
       <property name="maximum">
        <double>9999999.990000000223517</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="CkBResumen">
     <property name="text">
      <string>Hacer resumen</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>EDSBPrecio</class>
   <extends>QDoubleSpinBox</extends>
   <header>edsbprecio.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
